app/assets/stylesheets/variables.scss

```diff
@@ -8,7 +8,8 @@ $mid: #aaa;
 $accent: #F07C6C;
 $accent-light: #F39E8E;
 $dark: #111;
-
+$footerHeight: 237px;
+$negFooterHeight: -237px;
```

Comment:
For some reason, it didn't like me negating variables in the `margin: 0px auto -$footerHeight;` section 🤔

Reply:
Yeah sass limitations :( has to be a whole value and not a part of a value I guess